### PR TITLE
Add support for SignOffs - avskrivning

### PR DESCRIPTION
**Pull request by dkumanov**
**Expected Approvals: 2**

### CODE REVIEW

File: src/main/java/com/documaster/rms/noark/ws/noarkentities/Avskrivning.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+		return link(JOURNALPOST_LINK, journalpost);
+	}
+
+	public LinkAction<Avskrivning> linkTilknyttetJournalpost(String... journalpostIds) {
+
+		return link(TILKNYTTET_JOURNALPOST_LINK, journalpostIds);
+	}
+
+	public LinkAction<Avskrivning> linkTilknyttetJournalpost(Avskrivning... journalposts) {
```
Comment (ivaylomitrev) [inline src/main/java/com/documaster/rms/noark/ws/noarkentities/Avskrivning.java]: I think the type should be Journalpost instead.

Comment (ivaylomitrev) [reply src/main/java/com/documaster/rms/noark/ws/noarkentities/Avskrivning.java]: Same for unlink below.

**Comment by ivaylomitrev** [comment]
No other comments.

File: src/main/java/com/documaster/rms/noark/ws/noarkentities/Avskrivning.java (hunk 1 above)
Comment (dkumanov) [reply src/main/java/com/documaster/rms/noark/ws/noarkentities/Avskrivning.java]: Fixed with https://github.com/documaster/ws-client-n5-java/pull/32/commits/4162e287ca2f540de782346f12801a3f45827705

**Review by ivaylomitrev — APPROVED** [review]

**Review by plamenxpenchev — APPROVED** [review]

**Comment by dkumanov** [comment]
While testing the changes locally I discovered that the constructors for _Avskrivningsmaate_ (Sign Off Method) should be declared as public otherwise we will have issues with some serialization scenarios: [4c7fd6f](https://github.com/documaster/ws-client-n5-java/pull/32/commits/4c7fd6f540b314fd4b8699c489dcb0383cccfac4)

**Comment by ivaylomitrev** [comment]
> While testing the changes locally I discovered that the constructors for _Avskrivningsmaate_ (Sign Off Method) should be declared as public otherwise we will have issues with some serialization scenarios: [4c7fd6f](https://github.com/documaster/ws-client-n5-java/pull/32/commits/4c7fd6f540b314fd4b8699c489dcb0383cccfac4)

@dkumanov , I don't mind the change, but am wondering whether we need the JsonCreator constructor to be public as well as I see it's private for all entities we support.

**Comment by dkumanov** [comment]
> > While testing the changes locally I discovered that the constructors for _Avskrivningsmaate_ (Sign Off Method) should be declared as public otherwise we will have issues with some serialization scenarios: [4c7fd6f](https://github.com/documaster/ws-client-n5-java/pull/32/commits/4c7fd6f540b314fd4b8699c489dcb0383cccfac4)
> 
> @dkumanov , I don't mind the change, but am wondering whether we need the JsonCreator constructor to be public as well as I see it's private for all entities we support.

As far as I checked the JsonCreator constructor for the list value entities it is public, not private. The JsonCreator constructor is private for the "main" entities though.

**Review by ivaylomitrev — APPROVED** [review]

**Comment by dkumanov** [comment]
Squashed 3 commits into 1 in preparation for merge.

**Pull request merged**